src/content/indices/brazil.mdx

```diff
@@ -0,0 +1,151 @@
+---
+name: Brazil
+indexName: Índice de Qualidade do Ar 
+indexAcronym: IQA
+---
+
+import ColorScale from "@components/ColorScale.astro";
+import BreakpointsTable from "@components/BreakpointsTable.astro";
+import PiecewiseLatexDoc from "@components/PiecewiseLatexDoc.astro";
+import { AqiCalculator } from "@components/AqiCalculator.tsx";
+import Admonition from "@components/Admonition.astro";
+
+## Background
+
+Unlike some countries which operate under a unified national air quality index, Brazil employs municipal air quality indices, with some cities or regions developing their own system based on 
+local conditions, pollutants, and monitoring. Cities like Rio de Janeiro and São Paulo implement their own methodologies for IQA calculation. 
+
+In Rio de Janeiro, the Índice de Qualidade do Ar (IQA) is the official metric used to communicate daily air quality levels to the public. Adopted and defined by the State Environmental 
+Institute, the IQA follows standards consistent with those used by Monitorar-Rio, the state’s air quality monitoring program.
+
+Following a similar approach, Brazil's IQA serves as a public health communication tool, translating complex pollutant concentration data into a single numeric value on a scale from 0 to 300.
+
+Air quality is categorized into five qualitative levels, ranging from “Good” to “Terrible,” each indicating the potential health risks of exposure. A lower IQA value (e.g., 50 or below) 
+suggests clean air with minimal health risks, while values above 300 reflect severe pollution levels that pose serious health threats to the general population.
+
+This standardized index enables citizens, policymakers, and institutions in Rio de Janeiro to stay informed about current air quality conditions and respond appropriately to pollution-related 
+health risks.
+
+São Paulo uses a system established by the São Paulo State Environmental Agency (CETESB), which also features a five-tier scale but includes categories like “Extremely High Pollution.” 
+In contrast, Rio de Janeiro’s system, implemented by the Municipal Department of the Environment (SMAC), uses slightly different pollutant thresholds to define each air quality category. 
+These regional variations reflect differences in pollutants monitored, local climate, population sensitivity, and regulatory frameworks.
+
+
+## Color scale
+
+Brazil’s SMAC IQA calculation consists of five categories: 
```

Comment:
list out category names in orginal Brazlian Portuguese with english in parentheses